src/components/navigator-bar/navigator-bar.vue.ts

```diff
@@ -0,0 +1,21 @@
+import Vue from 'vue';
+import { Component } from 'vue-property-decorator';
+
+@Component
+export default class NavigatorBar extends Vue {
+    public async directToHome() {
+        await this.$router.push('/');
+    }
+    public async directToFunMath() {
+        await this.$router.push({name: 'Fun Math'});
+    }
+    public async directToAbout() {
+        await this.$router.push({name: 'About'});
+    }
+    public async directToProjects() {
+        await this.$router.push({name: 'Projects'});
+    }
+    public async directToMeme() {
+        await this.$router.push({name: 'Meme'});
+    }
```

Comment:
This is called code duplication. All of these functions can be combined into one:
```ts
public async directToPath(name: string) {
   await this.$router.push({name});
}
```

Also, we'd avoid having too many magic strings. While these routes are usually finite and can be put in constants. I would recommend creating a new file in the same folder called `app-route.ts` and have an enum in there:
```ts
export enum AppRoute {
   Home: 'Home',
   FunMath: 'Fun Math',
   ...
}
```